app/helpers/detailed-date.js

```diff
@@ -0,0 +1,12 @@
+import { helper } from '@ember/component/helper';
+import lightFormat from 'date-fns/lightFormat';
+
+export default helper(function detailedDate([datetime]/*, hash*/) {
+  try {
+    return lightFormat(datetime, 'dd/MM/yyyy HH:mm');
```

Comment:
I think this can be achieved with Intl.Datetimeformat as well by using some of the options:

`console.log(new Intl.DateTimeFormat('nl-BE', { dateStyle: 'short', timeStyle: 'short' }).format(date));`